e2e_deployment_files/stress_deployment_windows.template.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
Please add comment to mention issues about influxdb and snitcher modules.

nit: suggest to rename it as "stress_deployment.template.windows.json"